WordPress/gutenberg
# Gallery block: add an additional deprecation to fix blocks with missing data-full-url attrib

## Description
It seems like there is a missing deprecations as a Gallery block was encountered in the wild that has no `data-full-url` attrib when images are linked to file.

## Testing Instructions
- Try pasting the following into editor code view and make sure it migrates successfully:
```

<figure class="wp-block-gallery columns-2 is-cropped">
	<ul class="blocks-gallery-grid">
		<li class="blocks-gallery-item">
			<figure><a href="https://sergioestevaofolio.files.wordpress.com/2016/09/cropped-img_9054-1.jpg"><img class="wp-image-1421" src="https://sergioestevaofolio.files.wordpress.com/2016/09/cropped-img_9054-1.jpg?w=190" alt="" data-id="1421" data-link="https://sergioestevaofolio.files.wordpress.com/2016/09/cropped-img_9054-1.jpg" /></a></figure>
		</li>
		<li class="blocks-gallery-item">
			<figure><a href="https://sergioestevaofolio.files.wordpress.com/2017/09/cropped-l1001498-1.jpg"><img class="wp-image-1440" src="https://sergioestevaofolio.files.wordpress.com/2017/09/cropped-l1001498-1.jpg?w=580" alt="" data-id="1440" data-link="https://sergioestevaofolio.files.wordpress.com/2017/09/cropped-l1001498-1.jpg" /></a></figure>
		</li>
	</ul>
</figure>

```
 - Make sure `npm run test-unit` runs
